dev/Common/Microsoft.WindowsAppSDK.SelfContained.h

```diff
@@ -0,0 +1,48 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#ifndef __MICROSFT_WINDOWSAPPSDK_SELFCONTAINED_H
+#define __MICROSFT_WINDOWSAPPSDK_SELFCONTAINED_H
+
+//TODO:37999189 Support 3rd scenario (Yes=DevInnerLoop, Yes=PipelineBuild product, No=PipelineBuild Unit/FunctionalTests)
+#if defined(WINDOWSAPPSDK_SELFCONTAINED_RUNTIME_PACKAGE_FRAMEWORK_PACKAGEFAMILYNAME_W)
```

Comment:
why is SELFCONTAINED in the symbol?

Reply:
Don't sweat it this is going away. It was a quick hack to unblock some testing but needs to change from inline to an exported function to support all 3 scenarios.